src/Summoner/Tui.hs

```diff
@@ -173,6 +179,10 @@ app = App
         VtyEvent V.EvResize {}       -> continue s
```

Comment:
`s`?

Reply:
`s` has type `Form SummonKit e SummonForm`. Maybe `form` is better name...

Reply:
`formState` or `formApp`?